### ISSUE TITLE
[Bug]: Flatpak runtime org.kde.Platform branch 6.4 is end-of-life

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#2296]
### JASP Version

0.18.0

### JASP Module

Unrelated

### What OS are you seeing the problem on?

Flatpak

### Bug Description

Info: runtime org.kde.Platform branch 6.4 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Info: applications using this runtime:
   org.jaspstats.JASP

### Expected Behaviour

The packaged runtime should not be deprecated

### Steps to Reproduce

1. install jasp flatpak
2. flatpak update -y

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by RensDofferhoff** [comment]
Hi thanks for your report.
We will release an upcoming hotfix on this version.
After that we will move to a new version.
Testing everything takes a lot of work especially for minor releases.

**Comment by btreut** [comment]
@RensDofferhoff any news about this issue. I also get these error messages:

> Info: org.kde.Platform//6.4 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Applications using this runtime:
   org.jaspstats.JASP

**Comment by shun2wang** [comment]
>any news about this issue. I also get these error messages:

These is not any application and analysis errors but some flatpak warnnings, we had move to Qt6.5 so will disappeare in next release.

**Comment by kde-user** [comment]
I get this message too with featherwallet:
```
 ~  flatpak update                                                                                                  ✔ 
Check for updates …

Info: runtime org.kde.Platform branch 6.4 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Info: applications using this runtime:
   org.featherwallet.Feather

Nothing to do.
```

**Comment by tomtomme** [comment]
fixed with current 0.18.2 beta

**Comment by btreut** [comment]
Thanjs
